migrations/sqls/20221006114527-init-up.sql

```diff
@@ -0,0 +1,21 @@
+CREATE SCHEMA jira;
+
+CREATE TABLE jira.project (
+  id VARCHAR(10) PRIMARY KEY,
+  name VARCHAR(255) NOT NULL
+);
+
+CREATE TABLE jira.issue (
+  id VARCHAR(10) PRIMARY KEY,
+  project_id VARCHAR(10) NOT NULL,
```

Comment:
does this reference the project table ? should there be a fk constraint?

Reply:
It does, but the project and issue data is imported at the same time so I didn't want to make a hard constraint as the project might not have been imported at the time the issue gets imported.